### DISCUSSION TITLE
Boosted Random Forest (Final Project)

### DISCUSSION THREAD

**Discussion by micontre** [williamgilpin/cphy#14] [projects]
Post info here

**Comment by williamgilpin** [comment]
https://xgboost.readthedocs.io/en/stable/

https://lightgbm.readthedocs.io/en/v3.3.2/

**Reply by williamgilpin** [reply]
https://hastie.su.domains/Papers/ESLII.pdf

**Comment by micontre** [comment]
https://arxiv.org/pdf/physics/0408124.pdf

**Reply by williamgilpin** [reply]
2004? Impressive

**Comment by domstec** [comment]
https://towardsdatascience.com/decision-tree-classifier-explained-in-real-life-picking-a-vacation-destination-6226b2b60575

https://towardsdatascience.com/random-forests-algorithm-explained-with-a-real-life-example-and-some-python-code-affbfa5a942c

**Reply by domstec** [reply]
https://metapdf.hepforge.org/PDFSense/

**Comment by micontre** [comment]
Hi William, do you have an example of continuous data that we could use?

**Reply by williamgilpin** [reply]
let me think about it, a time series seems like a good potential option. perhaps an ECG trace from PhysioNet?

**Reply by williamgilpin** [reply]
@micontre I have a couple of time series datasets here: https://github.com/williamgilpin/fnn/tree/master/datasets

Numpy can read these files into arrays
